packages/shared/lib/services/sync/config/deploy.service.ts

```diff
@@ -658,59 +588,16 @@ export async function deployPreBuilt({
             await switchActiveSyncConfig(id);
         }
 
-        let content;
         const names = configs.map((config) => config.name || config.syncName);
-        if (isPublic) {
-            content = `Successfully deployed the ${nameOfType}${configs.length === 1 ? '' : 's'} template${configs.length === 1 ? '' : 's'} (${names.join(
-                ', '
-            )}).`;
-        } else {
-            content = `There ${configs.length === 1 ? 'was' : 'were'} ${configs.length} ${nameOfType}${configs.length === 1 ? '' : 's'} private template${
-                configs.length === 1 ? '' : 's'
-            } (${names.join(', ')}) deployed to your account by a Nango admin.`;
-        }
 
         await logCtx.info('Successfully deployed', { nameOfType, configs: names });
         await logCtx.success();
 
-        await telemetry.log(
-            LogTypes.SYNC_DEPLOY_SUCCESS,
-            content,
-            LogActionEnum.SYNC_DEPLOY,
-            {
-                environmentId: String(environment.id),
-                syncName: configs.map((config) => config.name).join(', '),
-                accountId: String(account.id),
-                integrations: configs.map((config) => config.provider).join(', '),
-                preBuilt: 'true',
-                is_public: isPublic ? 'true' : 'false'
-            },
-            `deploy_type:${isPublic ? 'public.' : 'private.'}template`
-        );
-
         return { success: true, error: null, response: { result: flowReturnData, logCtx } };
     } catch (err) {
-        const content = `Failed to deploy the ${nameOfType}${configs.length === 1 ? '' : 's'} (${configs.map((config) => config.name).join(', ')}).`;
-
         await logCtx.error('Failed to deploy', { nameOfType, configs: configs.map((config) => config.name), error: err });
         await logCtx.failed();
 
-        await telemetry.log(
-            LogTypes.SYNC_DEPLOY_FAILURE,
-            content,
-            LogActionEnum.SYNC_DEPLOY,
-            {
-                environmentId: String(environment.id),
-                syncName: configs.map((config) => config.name).join(', '),
-                accountId: String(account.id),
-                integration: configs.map((config) => config.provider).join(', '),
-                preBuilt: 'true',
-                is_public: isPublic ? 'true' : 'false',
-                level: 'error'
-            },
-            `deploy_type:${isPublic ? 'public.' : 'private.'}template`
-        );
-
```

Comment:
the customers deployments widget in the health dashboard depends on those SYNC_DEPLOY_SUCCESS/SYNC_DEPLOY_FAILURE logs I think

Reply:
Let's add a span

Reply:
we can use the express span I guess